nbs/API/frontmatter.ipynb

```diff
@@ -17,8 +17,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        def begin(self): self.fm = getattr(self.nb, 'frontmatter_', {})
</pre><p>This particular change was so that this processor doesn't completely delete the existing front matter if it happens to be there already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>